distribution/kubeflow/pipelines/patches/remove-mysql-resources.yaml

```diff
@@ -0,0 +1,37 @@
+#this will remove all resources related to on-cluster MySQL
+$patch: delete
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mysql
+---
+$patch: delete
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mysql-pv-claim
+---
+$patch: delete
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysql
+---
+$patch: delete
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: mysql
+  namespace: kubeflow
+---
+# $patch: delete
+# apiVersion: security.istio.io/v1beta1
+# kind: AuthorizationPolicy
+# metadata:
+#   name: mysql
```

Comment:
Is this re-used for RDS?

Reply:
No, but it seems to already have been removed somewhere in an earlier kustomization step. Apropos "eschewed feature"